config/locales/en.yml

```diff
@@ -54,6 +54,10 @@ en:
         crop_plantings: "Everyone's %{crop} plantings"
         owner_plantings: "%{owner} plantings"
 
+  photos:
+    show:
+      thing_by: "A %{thing} by %{owner}"
```

Comment:
This will be incorrect if `thing` starts with a vowel sound, but let's cross that bridge if we come to it :-)